File: app/src/main/java/de/xikolo/events/DownloadDeletedEvent.java
package de.xikolo.events;

import de.xikolo.events.base.Event;
import de.xikolo.utils.DownloadUtil;

public class DownloadDeletedEvent extends Event {

    public DownloadUtil.AssetDownload download;

    public DownloadDeletedEvent(DownloadUtil.AssetDownload download) {
        super(DownloadCompletedEvent.class.getSimpleName() + ": filePath = " + download.getFilePath() + ", type = " + download.getAssetType().toString());
        this.download = download;
    }

}

File: app/src/main/java/de/xikolo/events/DownloadDeletedEvent.kt
package de.xikolo.events

import de.xikolo.events.base.Event
import de.xikolo.models.DownloadAsset

class DownloadDeletedEvent(val downloadAsset: DownloadAsset) : Event(
    "${DownloadDeletedEvent::class.java.simpleName}: filePath = ${downloadAsset.filePath}, type = ${downloadAsset.title}"
)
